docs/developer/getting-started.md

```diff
@@ -82,6 +82,12 @@ Open a browser and access the UI under `localhost:9090`. The following processes
 
 In order to package everything into a ready-to-run Docker image, use the following task:
```

Comment:
This part should be after `export GOOS=linux`

Reply:
ah yeah that makes more sense, updated